test/teos/e2e/test_basic_e2e.py

```diff
@@ -88,13 +99,22 @@ def test_commands_registered(run_bitcoind):
     assert r.get("locator") == appointment.locator
     assert r.get("appointment") == appointment.to_dict()
     appointments_in_watcher += 1
+    available_slots -= 1
+
+    # Get subscription info
+    r = get_subscription_info()
+    assert r.get("appointments")[0].get("locator") == appointment.locator
```

Comment:
Once you fix the issue with the return, `.get("localtor")` won't be needed here.